tests/infra/consortium.py

```diff
@@ -348,10 +348,7 @@ def check_for_service(self, remote_node, status, pbft_open=False):
         """
         # When opening the service in PBFT, the first transaction to be
```

Comment:
If we're removing the PBFT-specific timeout, I guess this comment should go too?

Reply:
Oh my bad, we still have it on the `.post` call below! Cool.